lib/seedQuestions/index.js

```diff
@@ -587,12 +610,12 @@ class QuestionSeeder {
 }
 
 if (!module.parent) {
-    let addAllQuestionsFlag = QuestionSeeder.getAddAllQuestionsFlag();
-    let updateMdFlag = QuestionSeeder.getUpdateMDFilesFlag();
-
+    let addAllQuestionsFlag = QuestionSeeder.getAddAllQuestionsFlag()
+    let updateMdFlag = QuestionSeeder.getUpdateMDFilesFlag()
+    let updateToConfigFlag = QuestionSeeder.getUpdateConfigFileFlag()
     // both the question addition and update MD flag cannot be active at the same time
-    if (addAllQuestionsFlag && updateMdFlag) {
-        QuestionSeeder.showErrorAndExit("Both --addAllQuestions & --updateMD cannot be active at once.");
+    if (addAllQuestionsFlag && updateMdFlag && updateToConfigFlag) {
```

Comment:
If the user is using both --updateConfigFile & --ignoreConfigBucketsAndChoices simultaneously he should be given a warning and asked to confirm.

And you have missed `--ignoreConfigBucketsAndChoices` flag.

Reply:
I have added *--ignoreConfigBucketsAndChoices* flag but not understood well.